backend/infrahub/graphql/queries/account.py

```diff
@@ -29,7 +29,7 @@ class AccountTokenEdge(ObjectType):
 
 class AccountTokenEdges(ObjectType):
     count = Field(Int, required=True)
-    edges = Field(List(of_type=AccountTokenEdge, required=True), required=True)
+    edges = Field(List(of_type=AccountTokenEdge, required=True), required=False)
```

Comment:
This is a bit surprising to me. I might be incorrect here but the required=True part should indicate that `edges` has to be an array/list. Meaning that an empty list would be fine but a `null` value would not. Here we have errors in the rest of the schema where this isn't the case.

I.e. to me changing required to False would mean that both `null` and `[]` are valid responses but the API contract is cleaner if only `[]` is allowed (or a response that contains results.

If this change fixes anything I think it would be good to understand what is happening in the frontend.

Reply:
I guess the frontend was happy with the null list that was returned when no tokens were found. I changed this a bit by always returning a list (non-null) instead of a null value.